jenkins/lava-boot-v2.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+# -*- coding:utf-8 -*
+#
+
+import os
+from subprocess import Popen
+
+labs = {
+    'lab-free-electrons': {
+        'blacklist': [
+            'android'
+        ],
+    },
+}
+
+# Basic env
+LAB = os.environ['LAB']
+API = os.environ['API']
+API_TOKEN = os.environ['API_TOKEN']
+STORAGE = os.environ['STORAGE']
+CALLBACK = os.environ['CALLBACK']
+GIT_DESCRIBE = os.environ['GIT_DESCRIBE']
+TREE = os.environ['TREE']
+BRANCH = os.environ['BRANCH']
+ARCH = os.environ['ARCH']
+
+# Tokens
+LAVA_TBAKER_TOKEN = os.environ['LAVA_TBAKER_TOKEN']
+LAVA_MHART_TOKEN = os.environ['LAVA_MHART_TOKEN']
+LAVA_COLLABORA_TOKEN = os.environ['LAVA_COLLABORA_TOKEN']
+LAVA_BAYLIBRE_TOKEN = os.environ['LAVA_BAYLIBRE_TOKEN']
+LAVA_BAYLIBRE_SEATTLE_TOKEN = os.environ['LAVA_BAYLIBRE_SEATTLE_TOKEN']
+LAVA_FREE_ELECTRONS_TOKEN = os.environ['LAVA_FREE_ELECTRONS_TOKEN']
+
+# Command making the jobs
+make_jobs_args = [
```

Comment:
I also agree, but it's a bit more rework. The primary goal was to add per lab filtering on the trees. @mattface told me that a simple rework of the bash script would do the job for now (or I misunderstood, that possible too :D)

In any case, that would surely be a good idea not to spread configuration across too many files.